recipes/poliastro/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install --no-deps .
```

Comment:
I've seems some recipes like that in the wild and I wonder what are the possible problems.